pyvcloud/vcd/org.py

```diff
@@ -107,6 +107,28 @@ def get_catalog(self, name):
                 return self.client.get_resource(link.href)
         raise Exception('Catalog not found.')
 
+    def update_catalog(self, old_catalog_name, new_catalog_name, description):
```

Comment:
Can you add docstring for this method.

Reply:
I added a comment before the method. Is that what you meant by "docstring"?

Reply:
No. it has to be after the method name. You can follow the docstring format present in pyvcloud/vdc.py

Reply:
OK, done.